recipes/cuda/meta.yaml

```
@@ -0,0 +1,37 @@
+{% set version = "12.0.0" %}
+
+package:
+  name: cuda
+  version: {{ version }}
+
+source:
+  url: https://developer.download.nvidia.com/compute/cuda/redist/cuda_documentation/LICENSE.txt
+  sha256: 5db25d4fd138013b563f9a3d1d87f7de7df1dac014fd4cccdfbb3435a5cff761
+
+build:
+  number: 0
+  skip: true  # [osx or win]
+
+requirements:
+  run:
+    - cuda-runtime {{ version }}
+    - cuda-toolkit {{ version }}
+
+test:
+  commands:
+    - exit 0
+
+about:
+  home: https://developer.nvidia.com/cuda-toolkit
+  license_file: LICENSE.txt
+  license: LicenseRef-NVIDIA-End-User-License-Agreement
+  license_url: https://docs.nvidia.com/cuda/eula/index.html
+  summary: Meta-package containing all the available packages for native CUDA development
+  description: |
+    Meta-package containing all the available packages required for native CUDA
+    development
+  doc_url: https://docs.nvidia.com/cuda/index.html
+
+extra:
+  recipe-maintainers:
+    - adibbley
```

Let's go ahead and update this to include all the people we want here. Would suggest the following

```suggestion
extra:
  recipe-maintainers:
    - adibbley
    - bdice
    - jakirkham
    - leofang
    - vyasr
```

---

cc @kmittman